app/src/main/java/mozilla/lockbox/view/SettingFragment.kt

```diff
@@ -23,6 +38,28 @@ class SettingFragment : BackableFragment(), SettingViewProtocol {
         presenter = SettingPresenter(this)
 
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_setting, container, false)
+        val view = inflater.inflate(R.layout.fragment_setting, container, false)
+
```

Comment:
Nit: Set `android:overScrollMode="ifContentScrolls"` in the `RecyclerView` xml please.